src/StackExchange.Redis/ResultProcessor.cs

```diff
@@ -2068,6 +2069,34 @@ protected override bool SetResultCore(PhysicalConnection connection, Message mes
             }
         }
 
+        private sealed class TieBreakerProcessor : ResultProcessor<string>
+        {
+            protected override bool SetResultCore(PhysicalConnection connection, Message message, in RawResult result)
+            {
+                switch (result.Type)
+                {
+                    case ResultType.SimpleString:
+                    case ResultType.BulkString:
+                        var tieBreaker = result.GetString();
+                        SetResult(message, tieBreaker);
+
+                        var bridge = connection.BridgeCouldBeNull;
+                        try
+                        {
+                            var endpoint = bridge?.ServerEndPoint;
+                            if (endpoint != null)
```

Comment:
nit: `is not null`

Reply:
If we're getting cheeky, we're going all the way:
```cs
if (connection.BridgeCouldBeNull?.ServerEndPoint is ServerEndPoint endpoint)
```